src/network/clientopcodes.cpp

```diff
@@ -150,9 +150,9 @@ const ServerCommandFactory serverCommandFactoryTable[TOSERVER_NUM_MSG_TYPES] =
 	null_command_factory, // 0x14
 	null_command_factory, // 0x15
 	null_command_factory, // 0x16
-	null_command_factory, // 0x17
-	null_command_factory, // 0x18
-	null_command_factory, // 0x19
+	{ "TOSERVER_MODCHANNEL_JOIN", 0, true }, // 0x17
+	{ "TOSERVER_MODCHANNEL_LEAVE", 0, true }, // 0x18
```

Comment:
couldn't these be unified into a single `TOSERVER_MODCHANNEL_SIGNAL`?

Reply:
signal is a pure signal ID to define an ack or unack in the channel protocol

Reply:
the signal packet serves a different purpose indeed
nonetheless these two could be compacted into a single pkt

Reply:
it's possible but i don't see any interest, i prefer to have two command ids for it, like we do in every ADD/REMOVE packet.
Signal is now used to transmit some channel flags too